### PR TITLE
INS-27732 OpenSearch 1.x Patching and Bumping

### PR DESCRIPTION
**Pull request by tan3-netapp**

### CODE REVIEW

**Comment by chengwushi-netapp** [comment]
It's nice that you are including updates from other bundle teams as well. Not sure if you need to inform the other teams that you have updated their doc. Really minor.

**Review by chengwushi-netapp — APPROVED** [review]
lgtm

File: docs/index.md [hunk 1]
```diff
@@ -11,6 +11,20 @@ A Terraform provider for managing resources on the [Instaclustr Platform](https:
 
 It provides a flexible set of resources for provisioning and managing Instaclustr based clusters with the use of Terraform.
 
+This provider is in __General Availability__ with support for the following offerings:
```
Comment (arjunrajnetapp) [inline docs/index.md:14]: This looks like an error from the previous time this block was meant to be removed. Could you remove this added block from `/docs-base/index.md` and re-run the generation script?

File: docs/index.md [hunk 2]
```diff
@@ -44,138 +58,3 @@ provider "instaclustr" {
 ## Migrating from v1 to v2 of the Instaclustr Terraform Provider
 
 With the v2 version of the Instaclustr Terraform Provider, new resources have been introduced and schemas of existing resources have been changed. While the [Terraform Registry](https://registry.terraform.io/providers/instaclustr/instaclustr/latest/docs) contains the schema definitions of each resource, for a tool assisted approach of migrating to the v2 version of the Terraform Provider, see our support article on [importing Terraform resources](https://www.instaclustr.com/support/api-integrations/integrations/terraform-code-generation/).
-
-## Known Limitations
```
Comment (arjunrajnetapp) [inline docs/index.md:48]: Similar to above, would need to be added to `/docs-base/index.md` to have 0 differences in this file. Probably best to check with the team that rolled out this change first to confirm with them and let them know first.

File: docs/index.md (hunk 1 above)
Comment (tan3-netapp) [reply docs/index.md:14]: Team Customer Platform has been aware of this. They will fix it in PR#373 soon. I will not include `docs-base/index.md` in this commit, and remove the change in `docs/index.md`.

File: docs/index.md (hunk 2 above)
Comment (tan3-netapp) [reply docs/index.md:48]: Same as above

**Review by arjunrajnetapp — APPROVED** [review]

**Review by iclonghoang — APPROVED** [review]

**Review by arjunrajnetapp — APPROVED** [review]

**Review by iclonghoang — APPROVED** [review]

**Review by isuru-instaclustr — APPROVED** [review]

**Review by yallen-ic — APPROVED** [review]
LGTM

**Pull request merged**